<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndNewFirefighter</class>
 <widget class="QMainWindow" name="wndNewFirefighter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>432</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New Firefighter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbxNewFirefighter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Add New Firefighter</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFirstName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMiddleName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLastName">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Birth:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateDob">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>8</month>
       <day>20</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Local ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLocalID">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStateID">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zip Code:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCity">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="txtState">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>32</number>
     </property>
     <item>
      <property name="text">
       <string>AL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NJ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WY</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtZipCode">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Join Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Home Phone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Work Phone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell Phone:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStatus">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtHphone">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtWphone">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCphone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Driver's License:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CDL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDrvLic">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCDL">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateJoin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>8</month>
       <day>20</day>
      </datetime>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>270</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Firefighter</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>wndNewFirefighter</receiver>
   <slot>btnAdd_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnAdd_Clicked()</slot>
 </slots>
</ui>
